File: test/java/fibonacci.java
import package pheno;
 import java.util.List;
 import java.util.ArrayList;
 import java.math.BigDecimal;
public class fibonacci
{
    public int value;
    public fibonacci(int _value)
    {
        this.value=_value;
    }
}

class Estimation_fibonacci
{
    public static fibonacci Calculatefibonacci(int n)
    {


/*
     fibonacci function

    Author: 
    Reference:  to write in package
    Instituton: INRA Montpellier
    Abstract: see documentation
    
*/
        int value;

        int b=1; 
        value=0;  
        for(int i=0;i<n;i++)  
        {    
            int temp = value;   
            value=b;    
            b=temp+b;    
        }
        return new fibonacci(value);
    }

}

File: crop2ml/algo/openalea/fibonacci.py
import numpy as np 
from copy import copy
from math import *

def fibonacci(n):
    """


    fibonacci function
    Author: Pierre Martre
    Reference:  to write in package
    Institution: INRA Montpellier
    Abstract: see documentation

    """
    value = 0
    b = 1
    for i in range(0,n):
        temp = value
        value = b
        b = temp + b
    return  value
